lib/foreman_probing_core.rb

```diff
@@ -0,0 +1,15 @@
+require 'dynflow'
+require 'foreman_tasks_core'
+
+module ForemanProbingCore
+  extend ForemanTasksCore::SettingsLoader
+
+  if ForemanTasksCore.dynflow_present?
```

Comment:
Should this be updated now? AFAIK Dynflow should always be present and we want to get rid of `ForemanTasksCore`, right?